lib/addons/s3-csi-driver/index.ts

```diff
@@ -0,0 +1,84 @@
+import { Construct } from "constructs";
+import { ClusterInfo } from "../../spi";
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { createNamespace, setPath, supportsALL } from "../../utils";
+import { getS3DriverPolicyStatements } from "./iam-policy";
+
+const S3_CSI_DRIVER_SA = 's3-csi-driver-sa';
+const S3_CSI_DRIVER = 's3-csi-driver';
+const S3_CSI_DRIVER_RELEASE = 's3-csi-driver-release';
+const S3_DRIVER_POLICY = 's3-csi-driver-policy';
+
+/**
+ * Configuration options for the add-on.
+ */
+export interface S3CSIDriverAddOnProps extends HelmAddOnUserProps {
+    /**
+     * ARN of the S3 bucket to be used by the driver
+     */
+    s3BucketArn: string;
```

Comment:
Should it be ARN or just bucket name? The policy attached will only allow buckets from the customer account.